aiida/calculations/transfer.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Implementation of Transfer CalcJob."""
+
+import os
+from aiida import orm
+from aiida.engine import CalcJob
+from aiida.common.datastructures import CalcInfo
+
+
+def validate_instructions(instructions, ctx):  # pylint: disable=inconsistent-return-statements, unused-argument
```

Comment:
```suggestion
def validate_instructions(instructions, _):
```

Reply:
Hahah, NOW I see what you meant in the [other PR](https://github.com/aiidateam/aiida-core/pull/4437#discussion_r526668141) 👍